dropwizard-jsonapi/src/main/java/io/rtr/jsonapi/JSONAPI.java

```diff
@@ -171,9 +171,13 @@ public ResourceObjectBuilder(ResponseData<D> data) {
 		
 		public ResourceObjectImpl<D> build() {
 			ResourceObjectImpl<D> doc = new ResourceObjectImpl<D>();
-			doc.setData(data);
 			doc.setType(getType(data.attributes));
-			
+			//if there are no attributes set, make the section null so it is hidden in the response
+			if(data.attributes.toString().equals("BaseModel{id='null', created=null, updated=null}")) {
```

Comment:
If the toString() method is changed (that is, a new field is added, or even if the spacing changes), this will fail.  I think that it'll be safer if we can just check for null on the id.  Are u able to just grab the attribute called "id" and check for null? I think that we can safely ignore other fields (like created, updated) since they don't really matter.